### ISSUE TITLE
[Question] Why avoided using Dynamic Routes while creating Server-side-Rendering-Landing-Page-with-Query-Params?

### ISSUE THREAD

**Issue by omareg94** [pkellner/pluralsight-course-server-side-rendered-react-nextjs#84]
Is there a reason why did you avoid using [Dynamic Routes](https://nextjs.org/docs/routing/dynamic-routes) in lesson [5.7. Server-side-Rendering-Landing-Page-with-Query-Params](https://github.com/pkellner/pluralsight-course-server-side-rendered-react-nextjs/tree/master/m5-Extending-Your-Server-side-Rendered-App-to-Real-World/07-Server-side-Rendering-Landing-Page-with-Query-Params) and created custom server.js while Dynamic Routes seems simpler?

**Comment by pkellner** [comment]
I agree that dynamic routes are simpler.  I probably picked query params instead because they give you a lot more flexibility but certainly, for my example you could use dynamic routing. In real world apps, I've found that it's easy to get boxed into a corner with dynamic routing.  Things like checking null conditions and conditionals around dynamic routing can be problematic.

**Comment by omareg94** [comment]
Thank you.
